packages/cli/test/simulation/multi_fork.test.ts

```diff
@@ -53,10 +53,11 @@ const env = SimulationEnvironment.initWithDefaults(
     },
   },
   [
-    {id: "node-1", cl: CLClient.Lodestar, el: ELClient.Geth, keysCount: 32, mining: true},
+    {id: "node-1", cl: CLClient.Lodestar, el: ELClient.Mock, keysCount: 32, mining: true},
```

Comment:
While fine to test now if you need, I would not run any Mock in the main test run. It breaks the realism of this sim test for no benefit. Best to add a second file that runs only with Mock to be taken over by eip-4844 runs

Reply:
Yes definitely, I was added here to test the changes on the CI.